projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -62,8 +64,10 @@ const Admin = () => {
 		if (
 			isModuleEnabled &&
 			isUpdatingJetpackSettings &&
```

Comment:
I think we can remove this check now

Reply:
I think we cant as then it would refresh just when the toggle is on. That means if it's on, it will instantly refresh on each page refresh

Reply:
Done in https://github.com/Automattic/jetpack/pull/34142/commits/4b5f559992941d4383d6aa1c16e78841ca89d8ea